satellite/marketing/service.go

```diff
@@ -78,14 +78,27 @@ func (s *Service) InsertNewOffer(ctx context.Context, offer *NewOffer) (o *Offer
 	return o, nil
 }
 
-// UpdateOffer modifies an existing offer in the db when the offer status is set to NoStatus
-func (s *Service) UpdateOffer(ctx context.Context, offer *UpdateOffer) (err error) {
+// RedeemOffer adds 1 to the number of redeemed for an offer
+func (s *Service) RedeemOffer(ctx context.Context, uo *UpdateOffer) (err error) {
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters

Reply:
Thanks for sharing the link. I used the named result bc I needed to pass the `error` into the monkit and that has been the patter for doing so. I can definitely change it

Reply:
no need to change I forget monkit's oddities